libnetwork/drivers/remote/driver_test.go

```diff
@@ -24,15 +24,10 @@ import (
 	"gotest.tools/v3/assert"
 )
 
-func decodeToMap(r *http.Request) (map[string]interface{}, error) {
-	var res map[string]interface{}
-	err := json.NewDecoder(r.Body).Decode(&res)
-	return res, err
-}
-
 func handle(t *testing.T, mux *http.ServeMux, method string, h func(map[string]interface{}) interface{}) {
 	mux.HandleFunc(fmt.Sprintf("/%s.%s", driverapi.NetworkPluginEndpointType, method), func(w http.ResponseWriter, r *http.Request) {
-		ask, err := decodeToMap(r)
+		var ask map[string]interface{}
+		err := json.NewDecoder(r.Body).Decode(&ask)
```

Comment:
Not a _new_ issue here, but this is one of those "this will silently ignore any leftovers in `r.Body`" situations, right?